kodi_addon_checker/xml_schema/gotham_metadata.xsd

```diff
@@ -8,12 +8,12 @@
         <xs:element name="description" type="translatedString" minOccurs="0" maxOccurs="unbounded"/>
         <xs:element name="disclaimer" type="translatedString" minOccurs="0" maxOccurs="unbounded"/>
         <xs:element name="platform" type="platformList" minOccurs="0" maxOccurs="1"/>
-        <xs:element name="language" type="nonEmptyString"  minOccurs="0" maxOccurs="1"/>
+        <xs:element name="language" type="xs:string"  minOccurs="0" maxOccurs="1"/>
         <xs:element name="license" type="nonEmptyString"  minOccurs="0" maxOccurs="1" />
-        <xs:element name="forum" type="nonEmptyString" minOccurs="0" maxOccurs="1"/>
-        <xs:element name="website" type="nonEmptyString" minOccurs="0" maxOccurs="1"/>
+        <xs:element name="forum" type="xs:string" minOccurs="0" maxOccurs="1"/>
+        <xs:element name="website" type="xs:string" minOccurs="0" maxOccurs="1"/>
```

Comment:
Wiki doesn't say that website can be left blank.

Reply:
Should be resolved